partiql-eval/src/env.rs

```diff
@@ -0,0 +1,65 @@
+use std::fmt::Debug;
+
+use partiql_value::{BindingsName, Tuple, Value};
+use unicase::UniCase;
+
+pub trait Bindings<T> {
+    fn get(&self, name: &BindingsName) -> Option<&T>;
+}
+
+impl Bindings<Value> for Tuple {
+    fn get(&self, name: &BindingsName) -> Option<&Value> {
+        match name {
+            BindingsName::CaseSensitive(s) => self.0.get(s),
+            BindingsName::CaseInsensitive(s) => {
+                //TODO
```

Comment:
Other than the fact that we should perhaps treat the case insensitive differently, what do you expect to be as part of `TODO`?

Reply:
That's the todo. It's not actually doing case sensitivity currently.